[ADD] new module website_event_attendee_with_login

chrisandrewmann opened the pull request:
Add new module website_event_attendee_with_login to require attendee registrants to have their own accounts.

All tickets will be under the name/email of the logged-in user preventing them from signing up members of the public.
This is important for companies who only allow approved members to attend events such as webinars.

chrisandrewmann committed:
Add new module website_event_attendee_with_login, to require event attendee registrants to have their own accounts.
All tickets will be under the name/email of the logged-in user preventing them from signing up members of the public.
This is important for companies who only allow approved members to attend events such as webinars.

chrisandrewmann committed:
Add blank README.rst

chrisandrewmann committed:
Add newline to README.rst

chrisandrewmann committed:
Split lines in model, too long

emagdalenaC2i commented:
There are the module website_event_require_login for Odoo 11 "https://github.com/OCA/event/tree/11.0/website_event_require_login"
You should migrate it instead of create a new one.

chrisandrewmann commented:
Hello there,
This is not the same module, it offers different functionality on the attendee registering for a ticket, preventing them from specifying different email/name to their logged-in user account.
website_event_require_login gives different functionality and you can speak to the owners of that module who rejected my suggestion to add this functionality to their module.
Regards,
Chris

chrisandrewmann commented:
Please see this previous PR which explains all:
[https://github.com/OCA/event/pull/164](https://github.com/OCA/event/pull/164)

Commit:
Add new features to website_registration_template_views.
Hide register button if ticket already registered under this login account to prevent duplicates.

Also hide the select drop-down and force only 1 ticket to be allowed under this account, preventing duplicate registrations from non-approved accounts for events.
Show the total amount of available seats instead of the select drop-down field if not unlimited on the event.

Fix bug with website_event_registration_attendee_details_views, should check for false not 'false'.

chrisandrewmann committed:
Change use of class to hasclass in Qweb.
Add names to all Qweb template views.
Relocate tickbox on event form to right-hand column.

Commit:
Fix bug that causes crash when E-commerce is installed or "Tickets" and "Online Ticketing" is enabled

chienandalu commented on website_event_attendee_with_login/.gitignore:
```diff
@@ -0,0 +1,352 @@
+## Ignore Visual Studio temporary files, build results, and
```
Use repo .gitignore

chienandalu reviewed (commented):
Why not simply bypass the ticket form fill and get the logged in user for it? You could depend on website_event_require_login to do so. I'd rename the module to something like website_event_force_attendee_from_login as well.
